examples/image-to-text/README.md

```diff
@@ -30,4 +30,27 @@ python3 run_pipeline.py \
 Models that have been validated:
   - [nlpconnect/vit-gpt2-image-captioning](https://huggingface.co/nlpconnect/vit-gpt2-image-captioning)
   - [Salesforce/blip-image-captioning-large](https://huggingface.co/Salesforce/blip-image-captioning-large)
-  - [Salesforce/blip-image-captioning-base](https://huggingface.co/Salesforce/blip-image-captioning-base)
\ No newline at end of file
+  - [Salesforce/blip-image-captioning-base](https://huggingface.co/Salesforce/blip-image-captioning-base)
+
+## Single-HPU FP8 inference
+
+2-steps run:  
+1) Measuring Model
+```bash
+QUANT_CONFIG="<full-parent-path>/optimum-habana/examples/image-to-text/quantization_config/maxabs_measure.json" python3 run_pipeline.py \
```

Comment:
follow the format in text-generation README. do not provide command that can't be run by copy/paste

Reply:
Done